introductory-tutorials/broader-topics-and-ecosystem/intro-to-julia-DataFrames/02_basicinfo.ipynb

```diff
@@ -4,16 +4,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Introduction to DataFrames\n",
-    "**[Bogumił Kamiński](http://bogumilkaminski.pl/about/), May 23, 2018**"
+    "# Introduction to DataFrames"
```

Comment:
Can we put back Bogumił's name here? He did write these tutorials. Feel free to update the date to today's date, but we should probably leave the original author's name. We can also add your name too if that's the concern? THanks

Reply:
Bogumił should absolutely get credit for seeding the project, but it's weird to have a by-line on every individual file. You're free to add it back, but to me, keeping his name there implies that he wrote it all or personally vouches for it.
It also made me question whether to contribute, because it looks like he "owns" the code.

Reply:
...it feels like editing someone's blog entry rather than contributing to open code.